pkg/controllers/resourcechange/resourcechange_controller_test.go

```diff
@@ -109,7 +109,7 @@ func TestFindPlacementsSelectedDeletedResV1Alpha1(t *testing.T) {
 					Status: fleetv1alpha1.ClusterResourcePlacementStatus{
 						SelectedResources: []fleetv1alpha1.ResourceIdentifier{
 							{
-								Group:     "abd",
+								Group:     "xyz",
```

Comment:
This change is solely made to pass the codespeller check; apparently the program thinks `abd` as a misspelling.

Reply:
Same for the change below.